tests/integration/test_http_get_all.py

```diff
@@ -0,0 +1,93 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import utils
+
+
+def test_http_get_all(test_env, default_request, tmpdir):
+    """
+    Check HTTP GET_ALL works as expected.
+
+    Process:
+    * Create at least two requests and wait for them to complete
+    * Send request to check all requests in Cachito at the /api/v1/requests API endpoint
+    * Send requests to check all requests with filtering parameters
+    Checks:
+    * Check that response code is 200
+    * Check that filtering parameters are working properly (page, per_page, and state)
+    """
+    client = utils.Client(test_env["api_url"], test_env["api_auth_type"])
+    requests_amount = test_env["http_get_all"]["requests_amount"]
+    submitted_requests = []
+    for _ in range(requests_amount):
+        initial_response = client.create_new_request(
+            payload={
+                "repo": test_env["package"]["repo"],
+                "ref": test_env["package"]["ref"],
+                "pkg_managers": test_env["package"]["pkg_managers"],
+            },
+        )
+        completed_response = client.wait_for_complete_request(initial_response)
+        response_data = completed_response.data
+        submitted_requests.append(response_data.get("id"))
```

Comment:
`id` should always be there and I want the test to fail if it's not set, so let's access the key directly.
```suggestion
        submitted_requests.append(response_data["id"])
```

Reply:
Done, thank you.